packages/markdown/__tests__/index.test.js

```diff
@@ -96,13 +96,12 @@ test('should render nothing if nothing passed in', () => {
   expect(markdown('')).toBe(null);
 });
 
-test('`correctnewlines` option', () => {
```

Comment:
We still need this option to work. Is it incompatible with using rehype directly?